### DISCUSSION TITLE
Feature to report issues viewing content

### DISCUSSION THREAD

**Discussion by AliyahMillan** [darkreader/darkreader#7411] [Ideas]
On some websites, certain content won't show or content looks strange (see below for examples). 

I was hoping there could be a feature included in the extension that would allow users to report issues viewing content on websites (kind of like a feedback feature, but where users can include screenshots of missing content and the link to the website they're experiencing issues on).

_Feel free to reach out, as I'd be interested in seeing how I could be able to contribute to solving this issue!_  :grin:

![image](https://user-images.githubusercontent.com/60375020/142346450-3657a7ff-b1ce-4007-90cc-eb971c51f39b.png)
![image](https://user-images.githubusercontent.com/60375020/142346398-efe05180-7aba-4bc1-a671-d9369ca2b5ba.png)

**Comment by Gusted** [comment]
> I was hoping there could be a feature included in the extension that would allow users to report issues viewing content on websites (kind of like a feedback feature, but where users can include screenshots of missing content and the link to the website they're experiencing issues on).

What about users using the issues tab: https://github.com/darkreader/darkreader/issues/new/choose ?

**Reply by AliyahMillan** [reply]
I'd actually meant putting a button of sorts to do so just using the extension.
![image](https://user-images.githubusercontent.com/60375020/142688082-ba489323-18cc-43be-bc94-ba65c7d4d0b4.png)
Maybe somewhere along the bottom, just click a "Report Broken Website" button or something.

**Reply by Gusted** [reply]
@alexanderby
